docs/content/operator/common/monitoring.mdx

```diff
@@ -0,0 +1,123 @@
+After a successful setup of a node, the next important step is to set up monitoring for the node.
```

Comment:
No level one title and no front matter 😅

Reply:
The files in `common/` are shared-contents imported in files in `full-node/configs` and `validator-node/configs`.

Reply:
Thanks for the clarification. Makes sense ❤️